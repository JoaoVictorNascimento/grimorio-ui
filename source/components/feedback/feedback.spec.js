import '../../../internals/test/helper';

import Feedback from './feedback-component';

/** @test {Feedback} */
describe('Feedback component', () => {
  /** @test {Feedback#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(<Feedback />);
      expect(wrapper.length).toEqual(1);
    });
  });
});
