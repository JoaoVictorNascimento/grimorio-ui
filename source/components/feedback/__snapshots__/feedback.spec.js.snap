// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback component #render render when isOpen is false 1`] = `"<Symbol(react.fragment) />"`;

exports[`Feedback component #render render when isOpen is true 1`] = `
"<Symbol(react.fragment)>
  <div className=\\"defaultPosition\\">
    <div className=\\"default\\">
      <Icon className=\\"\\" size=\\"20\\" name=\\"error\\" align=\\"middle\\" />
      <span className=\\"message\\" />
      <Button className=\\"buttonClick\\" onClick={[Function: bound handleClick]} disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" active={false}>
        <Icon className=\\"iconCleanError\\" size=\\"22\\" name=\\"clear\\" align=\\"middle\\" />
      </Button>
    </div>
  </div>
</Symbol(react.fragment)>"
`;
